documentation/public/user-guide/09-appendix/02-slang-memory-management/index.md

```diff
@@ -0,0 +1,39 @@
+# 9.2. Considerations on memory usage
```

Comment:
I think there is a difference between the title `Considerations on memory usage` and the slug `slang-memory-management`. I suggest consolidating it here.